pkg/portal/cluster/prometheusalerts_test.go

```diff
@@ -0,0 +1,119 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/go-test/deep"
+	"github.com/golang/mock/gomock"
+	"github.com/prometheus/common/model"
+
+	mock_alertmanager "github.com/Azure/ARO-RP/pkg/util/mocks/alertmanager"
+	testlog "github.com/Azure/ARO-RP/test/util/log"
+)
+
+func TestFiringAlerts(t *testing.T) {
+	ctx := context.Background()
+	_, log := testlog.New()
```

Comment:
I don't think this should be used, you probably can just create logger that discard all entries

Reply:
Could you give more context on this

Reply:
```go
	logger := logrus.New()
	logger.SetOutput(io.Discard)
	entry := logrus.NewEntry(logger) //use that instead of the result of testlog.New()
```

Reply:
Done, created logger